src/viking.js

```diff
@@ -1,11 +1,82 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength =  strength;
+    }
+    attack() {
+        return this.strength;
+    }
+    receiveDamage(damage) {
+        let currentHealth = this.health - damage;
+    }
+}
+
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(health, strength, name){
+    super (health, strength)
+        this.name= name;
+    }
+    receiveDamage(damage){
+        let currentHealthViking = this.health - damage;
+        if (currentHealthViking > 0){return `${this.name} has received ${this.damage} points of damage`}
+        else if (currentHealthViking <= 0){return `${this.name} has died in act of combat`}
```

Comment:
Make use of `this.health` instead of using a new variable. Otherwise, the information on the output won't match

```suggestion
    this.health = this.health - damage;
    if (this.health > 0) {
      return `${this.name} has received ${damage} points of damage`;
    } else {
      return `${this.name} has died in act of combat`;
    }
```